context-free.md

```diff
@@ -2,21 +2,24 @@
 
 ##  Who is this programming language for?
 
+This language appears to be for people who want to create graphics programmatically (with repeated, recursive, random, etc. structures).
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to make basic shapes because the components are built in (lines, curves, circles, triangles, squares, etc.). A large advantage of this program is that it can iteratively create shapes which makes patterns easier. It can also recursively create shapes which makes complex structures like fractals relatively straightforward to draw (although it does need a minimum resolution, so I guess it's not quite a fractal).
 
 ## What is hard to do in this language? Why is it hard?
 
+It is difficult to draw realistic images (e.g. a face) since every part of every shape needs to be defined. Unlike in traditional drawing programs or on paper, in this program, you need to know the coordinates of everything.
 
 ## How did you learn how to program in this language?
 
+I read through parts of the ContextFree [wiki](https://github.com/MtnViewJohn/context-free/wiki).
```

Comment:
This is a great resource, I wish I had found this when learning how to use context free